src/Polytopes/Polyhedron/constructors.jl

```diff
@@ -60,43 +58,56 @@ pm_polytope(P::Polyhedron) = P.pm_polytope
 
 ### Construct polyhedron from V-data, as the convex hull of points, rays and lineality.
 @doc Markdown.doc"""
-    convex_hull(V [, R [, L]])
+    convex_hull(V::Matrix [, R::Matrix [, L::Matrix]]; non_redundant::Bool = false)
+
+Compute the convex hull of the rows of the matrix `V`. Optionally one can give
```

Comment:
Julia documenting conventions suggests to leave out redundancy where possible. This means not to state in the description that optional arguments are optional, if we can make this visible within the header in the docstring. Here, the square brackets do this job.